src/main/java/com/powsybl/openloadflow/network/VoltageControl.java

```diff
@@ -0,0 +1,44 @@
+package com.powsybl.openloadflow.network;
+
+import java.util.*;
+
+/**
+ * @author Anne Tilloy <anne.tilloy at rte-france.com>
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+public class VoltageControl {
+
+    private final LfBus controlled;
+
+    private final Set<LfBus> controllers;
+
+    private final double targetValue;
+
+    public VoltageControl(LfBus controlled, double targetValue) {
+        this.controlled = controlled;
+        this.targetValue = targetValue;
+        this.controllers = new LinkedHashSet<>();
+    }
+
+    public double getTargetValue() {
+        return targetValue;
+    }
+
+    public LfBus getControlledBus() {
+        return controlled;
+    }
+
+    public Set<LfBus> getControllerBuses() {
+        return controllers;
+    }
+
+    public void addControllerBus(LfBus controllerBus) {
+        Objects.requireNonNull(controllerBus);
+        controllers.add(controllerBus);
+        controllerBus.setVoltageControl(this);
+    }
+
+    public boolean isVoltageControlLocal() {
```

Comment:
The name is not clear, at first read I was thinking it means this is only a local control

Reply:
Indeed, but in fact we call it only once, to check just after if it is only local! hence I changed its content to return whether it is a local control only.